src/data/mv_polynomial/rename.lean

```diff
@@ -43,34 +43,34 @@ open set function finsupp add_monoid_algebra
 open_locale big_operators
 
 universes u v w x
-variables {α : Type u} {β : Type v} {γ : Type w} {δ : Type x}
+variables {R : Type u} {S : Type*} {β : Type v} {γ : Type w} {δ : Type x}
```

Comment:
mainly pointing out that this doesn't fit the pattern, but I'm not certain what `Type*` does so I'll leave it up to you.

Reply:
@shingtaklam1324 for you information `Type*` should be thought of as `Type _`, and it will create a fresh universe variable if it doesn't unify with an existing one.

Reply:
Thanks :) It just stood out to me since the rest of the variables are in one of the universes declared above.